src/App.vue

```diff
@@ -302,12 +300,18 @@ export default class App extends Mixins(
   /** The Update Current JS Date timer id. */
   private updateCurrentJsDateId = 0
 
+  /** Returns URL param string with Account ID if present, else empty string. */
+  private getParams (): string {
+    const accountId = sessionStorage.getItem('ACCOUNT_ID')
+    return accountId ? `?accountid=${accountId}` : ''
+  }
+
   /** The route breadcrumbs list. */
   private get breadcrumbs (): Array<BreadcrumbIF> {
     const crumbs: Array<BreadcrumbIF> = [
       {
         text: this.legalName || this.getNumberedEntityName,
-        href: `${sessionStorage.getItem('DASHBOARD_URL')}${this.getEntityIdentifier}`
+        href: `${sessionStorage.getItem('DASHBOARD_URL')}${this.getEntityIdentifier}/${this.getParams()}`
       },
```

Comment:
I'm thinking of moving this breadcrumb to BreadCrumbResources.ts to get rid of the duplicate `getParams()` above, but on the other hand I'd have to pass in legalName, getNumberedEntityName and getEntityIdentifier. Thoughts?

Reply:
Fine with whatever, should work as long as our resources are localized.
Is it easy to bind Getters in a resource file?

Reply:
It's possible, yes. For example:
```
import { getVuexStore } from '@/store'
const store = getVuexStore()
const getFilingType = store.getters.getFilingType as FilingTypes
```

I'm not sure which I prefer. Or rather, dislike the least 😛

Reply:
haha i hear ya. 
Your call!

Reply:
Well, have a look at my new commit and tell me what you think.

Reply:
Looks fine to me!

Reply:
OK. If we end up hating this code then we can change it later.